Don't assume that LD_LIBRARY_PATH is set, since this won't be the case
on a properly configured system, where all the library paths are
configured via the linker.

--- unoconv.~1~	2010-12-13 11:45:27.000000000 +0100
+++ unoconv	2010-12-13 11:45:45.000000000 +0100
@@ -64,7 +64,7 @@
             sys.path.append(oolibpath)
             import uno, unohelper
             ### Export an environment that OpenOffice is pleased to work with
-            os.environ['LD_LIBRARY_PATH'] = oolibpath + os.pathsep + os.environ['LD_LIBRARY_PATH']
+            os.environ['LD_LIBRARY_PATH'] = oolibpath + os.pathsep + os.environ.get('LD_LIBRARY_PATH', '')
             break
         except ImportError, e:
             sys.path.remove(oolibpath)
